modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -9,8 +9,6 @@ class Representative < ApplicationRecord
       BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
 
```

Comment:
Something like this to adhere to strong_migrations recommended process

```suggestion
self.ignored_columns += %w[dob_ciphertext encrypted_kms_key ssn_ciphertext]
```